soc/silabs/silabs_siwx917/siwg917/Kconfig.soc

```diff
@@ -19,3 +19,34 @@ config SOC
 
 config SOC_PART_NUMBER
 	default "SIWG917M111MGTBA" if SOC_PART_NUMBER_SIWG917M111MGTBA
+#Multiple 917 components depends on NWP and on CMSIS OS 2 thus a common default config here makes sense
+config SIWX917_NWP_NETWORK_STACK
```

Comment:
You can drop the comment and add an empty line between this config item and the previous one.